diff -rurN debian/changelog debian-new/changelog
--- debian/changelog	2007-05-03 07:15:49.000000000 +0200
+++ debian-new/changelog	2009-04-11 12:26:48.000000000 +0200
@@ -1,4 +1,14 @@
-wifidog (1.0.0-1) stable; urgency=low
+nodogsplash (0.9.9-3) stable; urgency=low
+
+  * Fixed buffer overflow bug in ndsctl.
+  * Improved SIGCHLD handling in gateway.c and util.c.
+  * Added FirewallRulesets trusted-users and trusted-users-to-router. This permits customized access control for trusted users, as for authenticated users, with the same ruleset semantics, instead of always accepting all trusted packets.
+  * Added EmptyRuleSetPolicy directives for .conf file.  Permits optionally passing packets through to pre-existing firewall rules.
+  * updated debian rules (for building debian package)
+
+ -- Andraz Sraka <a@frubsd.org>  Sun, 11 Aug 2009 09:15:00 -0100
+
+nodogsplash (0.3.0-1) stable; urgency=low
 
   * New init.d file.
   * Inclu
@@ -7,7 +17,7 @@
 
  -- Guillaume Beaudoin <isf@soli.ca>  Sun, 29 Aug 2004 23:14:12 -0400
 
-wifidog (0.2.0-1) stable; urgency=low
+nodogsplash (0.2.0-1) stable; urgency=low
 
   * Initial Package
 
diff -rurN debian/rules debian-new/rules
--- debian/rules	2007-05-03 07:15:49.000000000 +0200
+++ debian-new/rules	2009-04-11 12:26:48.000000000 +0200
@@ -7,7 +7,7 @@
 build-stamp:
 	dh_testdir
 
-	./configure --prefix=/usr
+	./configure --prefix=/usr --sysconfdir=/etc
 	$(MAKE)
 
 	touch build-stamp
@@ -30,8 +30,8 @@
 
 	$(MAKE) DESTDIR=$(CURDIR)/debian/tmp install
 	mkdir -p $(CURDIR)/debian/tmp/etc
-	cp wifidog.conf $(CURDIR)/debian/tmp/etc
-	cp scripts/init.d/wifidog debian/wifidog.init
+	cp nodogsplash.conf $(CURDIR)/debian/tmp/etc
+	cp scripts/init.d/S65nodogsplash debian/nodogsplash.init
 
 # Build architecture-independent files here.
 binary-indep: build install
